srv/modules/runners/populate.py

```diff
@@ -925,6 +925,18 @@ def engulf_existing_cluster(**kwargs):
     local.cmd("*", "saltutil.sync_all")
     local.cmd("*", "state.apply", ["ceph.mines"], expr_form="compound")
 
+    # TODO: is there a saner way to get the master_minion at this point (when
+    # there might not yet be a master role assigned?)
+    master_minion = ""
+    for minion, master in local.cmd("*", "pillar.get", ["master_minion"]).items():
+        if master:
```

Comment:
What do you think - `if master and master != "_REPLACE_ME_":` ?  Might save a developer, that copied over their wip branch, some debug time :smile: